llvm/include/llvm/IR/DIBuilder.h

```diff
@@ -606,6 +606,35 @@ namespace llvm {
         PointerUnion<DIExpression *, DIVariable *> Allocated = nullptr,
         PointerUnion<DIExpression *, DIVariable *> Rank = nullptr);
 
+    /// Create debugging information entry for an array.
+    /// \param Scope          Scope in which this enumeration is defined.
+    /// \param Name           Union name.
+    /// \param File           File where this member is defined.
+    /// \param LineNumber     Line number.
```

Comment:
these 4 descriptions are differently aligned than the rest - could you line them up (one way or another)?

Reply:
Oops, sorry.